doc/source/whatsnew/v0.17.1.txt

```diff
@@ -42,3 +42,5 @@ Performance Improvements
 
 Bug Fixes
 ~~~~~~~~~
+
+- Bug in `squeeze` with zero length arrays (:issue:`11230`) & (:issue:`8999`)
```

Comment:
use double-backtickers (and say ``.squeeze()``)

list issues like: ``(:issue: `11230`, :issue:`8999`)``